src/Cech_complex/include/gudhi/MEB_filtration.h

```diff
@@ -60,12 +68,18 @@ void assign_MEB_filtration(Kernel&&k, SimplicialComplexForMEB& complex, PointRan
       FT r = k.squared_distance_d_object()(m, pu);
       if (exact) CGAL::exact(r);
       complex.assign_key(sh, cache_.size());
-      complex.assign_filtration(sh, std::max(cvt(r), Filtration_value(0)));
+      complex.assign_filtration(sh, max(cvt(r), Filtration_value(0)));
       cache_.emplace_back(std::move(m), std::move(r));
+    } else if (dim > ambient_dim) {
+      // The sphere is always defined by at most d+1 points
+      Filtration_value maxf = 0; // max filtration of the faces
+      for (auto face : complex.boundary_simplex_range(sh)) {
+	maxf = max(maxf, complex.filtration(face));
```

Comment:
Bad indentation (with  tab on this line), should be with spaces

Reply:
Done.